sylt-macro/src/lib.rs

```diff
@@ -115,30 +115,28 @@ pub fn extern_function(tokens: proc_macro::TokenStream) -> proc_macro::TokenStre
         .collect();
 
     let tokens = quote! {
-        #[sylt_macro::sylt_doc(#function, #doc , #( #matches ),*)]
-        #[sylt_macro::sylt_link(#link_name, #module)]
+        #[sylt_macro::sylt_doc(#link_name, #doc, #signature)]
+        #[sylt_macro::sylt_link(#link_name, #module, #signature)]
         pub fn #function (
             ctx: sylt_common::RuntimeContext
         ) -> ::std::result::Result<sylt_common::Value, sylt_common::error::RuntimeError>
         {
             use sylt_common::MatchableValue::*;
             use sylt_common::RustFunction;
             use sylt_common::Value::*;
-            use sylt_common::value::make_matchable;
+            use std::borrow::Cow;
```

Comment:
Won't this throw "unused import" warnings?

Reply:
Don't know - but we definitely don't need it